bigquery_etl/cli/query.py

```diff
@@ -860,7 +861,20 @@ def _run_query(
             # point to a public table it needs to be passed as parameter for the query
             query_arguments.append("--destination_table={}".format(destination_table))
 
-        with open(query_file) as query_stream:
+        # write rendered query to a temporary file;
+        # query string cannot be passed directly to bq as SQL comments will be interpreted as CLI arguments
+        tmp = tempfile.NamedTemporaryFile()
+        with tempfile.NamedTemporaryFile(prefix=tmp.name, mode="w+") as query_stream:
```

Comment:
Do we still need `tmp` here? We could either omit the `prefix` or make it something meaningful for potential debugging